boot.php

```diff
@@ -35,6 +35,13 @@
     }
 }
 
+// switch REDAXO to frontend mode before generating cache files
+rex_extension::register('PACKAGES_INCLUDED', function(rex_extension_point $ep) {
+    if (rex_be_controller::getCurrentPagePart(2) == 'generator') {
```

Comment:
Ach, der @staabm, dich hatte ich ganz verpasst auf die Reviewer-Liste zu setzen. Aber du siehst sowas ja eh immer selbst ;) Danke fürs Feedback!

Den EP brauchen wir, um den Schalter so früh zu setzen, dass blÖcks bereits im Frontend-Modus läuft. Auf der Generator-Page konnte ich den EP nicht nutzen, das geht offenbar nicht? Deshalb ist er in der `boot.php` und feuert nur, wenn die Subpage `generator` ist.
Aber du hast recht: Weil es im EP ist, wird der Code vermutlich auch auf anderen generator-Subpages durchlaufen, und wir müssten nochmal spezifischer prüfen.